cypress/integration/dib-travel/company-settings/travel-policy.spec.ts

```diff
@@ -0,0 +1,109 @@
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog ui-button button').click({ force: true });
+    cy.get('dib-dialog-wrapper dib-travel-policy-dialog').should('be.visible');
+  });
+
+  it('creates a new flight travel policy', () => {
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog .dib-select').select(tpd.travelPolicyTypeFlight);
+    cy.get('dib-travel-policy-dialog dib-input input[name=name]').type(tpd.travelPolicyName);
+    cy.get('dib-travel-policy-dialog dib-input input[name=numberOfDaysInAdvance]').type(tpd.numberOfDaysInAdvance);
+    cy.get('dib-travel-policy-dialog dib-checkbox').click({ multiple: true });
+    cy.get('dib-travel-policy-dialog dib-input input[name=budget]').type(tpd.travelPolicyBudget);
+    cy.get('dib-travel-policy-dialog dib-input input[name=budgetException]').type(tpd.travelPolicyBudgetExeption);
+    cy.get('dib-travel-policy-dialog dib-input input[name=flightDurationBudgetException]').type(
+      tpd.flightDurationBudgetException
+    );
+    cy.get('dib-travel-policy-dialog form .add-btn').click({ force: true });
+    cy.get('dib-list-item dib-flight-search-input input[data-placeholder=From]').type(tpd.flightFrom);
+    cy.get('.cdk-overlay-connected-position-bounding-box .mat-option-text').contains(tpd.flightFrom).click();
+    cy.get('dib-list-item dib-flight-search-input input[data-placeholder=To]').type(tpd.flightTo);
+    cy.get('.cdk-overlay-connected-position-bounding-box .mat-option-text').contains(tpd.flightTo).click();
+    cy.get('dib-travel-policy-dialog dib-list-item select').select(tpd.typeOfTrip);
+    cy.get('dib-list-item dib-input input[name=budget]').type(tpd.travelPolicyBudget);
+    cy.get('dib-travel-policy-dialog dib-input input[placeholder=Search]').type(tpd.employeeName);
+    cy.get('dib-travel-policy-dialog mat-checkbox').click();
+    cy.get('dib-travel-policy-dialog ui-button button').click({ force: true });
+    cy.get('dib-travel-policy dib-expandable-item .section__header__title').should('contain', tpd.travelPolicyName);
+  });
+
+  it('creates a new hotel travel policy', () => {
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog .dib-select').select(tpd.travelPolicyTypeHotel);
+    cy.get('dib-travel-policy-dialog dib-input input[name=name]').type(tpd.travelPolicyName);
+    cy.get('dib-travel-policy-dialog dib-input input[name=numberOfDaysInAdvance]').type(tpd.numberOfDaysInAdvance);
+    cy.get('dib-travel-policy-dialog star-rating').click();
+    cy.get('dib-travel-policy-dialog dib-input input[name=budget]').type(tpd.travelPolicyBudget);
+    cy.get('dib-travel-policy-dialog form .add-btn').click({ force: true });
+    cy.get('dib-travel-policy-dialog dib-location-search input[placeholder=City]').type(tpd.city);
+    cy.get('.light-background .pac-container').contains(tpd.city).click();
```

Comment:
What is the parent feature of this?

Reply:
`.cdk-overlay-container`, I will fix it.